utils.groovy

```diff
@@ -927,7 +937,7 @@ def build_and_push_image(params = [:]) {
     def digests = build_remote_image(params['arches'], params['src_commit'], params['src_url'], params['staging_repository'],
```

Comment:
can we delete `manifest_tag_staging` as an argument to this function. It's really not needed.

Reply:
Where?

Reply:
I think maybe I'll just open a new PR next week. We can go with this for now.